examples/all_pattern.http.go

```diff
@@ -41,7 +42,7 @@ func (h *AllPatternHTTPConverter) AllPattern(cb func(ctx context.Context, w http
 			if err != nil {
 				w.WriteHeader(http.StatusInternalServerError)
 				p := status.New(codes.Unknown, err.Error()).Proto()
-				switch r.Header.Get("Content-Type") {
+				switch contentType, _, _ := mime.ParseMediaType(r.Header.Get("Content-Type")); contentType {
```

Comment:
I just ignored the returned error because empty `contentType` will through `default` case.